vowpalwabbit/core/src/reductions/cb/cb_explore_adf_cover.cc

```diff
@@ -323,11 +323,12 @@ VW::LEARNER::base_learner* VW::reductions::cb_explore_adf_cover_setup(VW::setup_
   }
 
   auto* scorer = VW::LEARNER::as_singleline(base->get_learner_by_name_prefix("scorer"));
+  auto* cost_sensitive = VW::LEARNER::as_multiline(base->get_learner_by_name_prefix("cs"));
```

Comment:
Should add a check in case there is no cost sensitive learner in the stack

Reply:
it throws https://github.com/VowpalWabbit/vowpal_wabbit/blob/24f66f44a9020437cf9942392925bf7e9daea7b8/vowpalwabbit/core/include/vw/core/learner.h#L521

Reply:
ah okay